### ISSUE TITLE
$mdDialog.prompt is not a function

### ISSUE THREAD

**Issue by brenolf** [angular/material#7535]
I've just tried out the example shown in the docs page but I always get this error when trying to use prompt.

I've also inspected `$mdDialog` and it returns:

```js
Object {}
  alert: (arg)
  build: (arg)
  cancel: cancel(reason, options)
  confirm: (arg)
  destroy: destroyInterimElement(opts)
  hide: hide(reason, options)
  show: showInterimElement(opts)
  __proto__: Object
```

Nevertheless I've checked that `prompt` is listed as a preset in the source code, but I didn't go any further to figure out why it's not being added.

**Comment by soohoowoohoo** [comment]
+1 Running into the same issue

**Comment by DevVersion** [comment]
@soohoowoohoo The prompt is only available in `v1.1.0rc1`.

Everything works well in my demo.
http://codepen.io/DevVersion/pen/aNZXoY

**Comment by brenolf** [comment]
@DevVersion it worked here, but since its on the master doc, shouldn't there be a note somewhere telling that we should use `v1.1.0rc1` instead?

**Comment by DevVersion** [comment]
The docs (https://material.angularjs.org/latest/) clear say, which feature is included in which version.

![image](https://cloud.githubusercontent.com/assets/4987015/13714163/41ec1930-e7cd-11e5-9830-2aefdefdcc7f.png)

Also the changelog is saying, that the prompt, is only available in `v1.1.0rc1`
https://github.com/angular/material/blob/master/CHANGELOG.md

I'll close that issue, because everything works as expected. If you have any further questions, just let me know ;)
